bluebottle/files/serializers.py

```diff
@@ -5,21 +5,20 @@
 from rest_framework_json_api.relations import ResourceRelatedField
 from rest_framework_json_api.serializers import ModelSerializer
 
-from bluebottle.files.models import File
+from bluebottle.files.models import Document, Image
 
 
 class FileField(ResourceRelatedField):
     def get_queryset(self):
-        return File.objects.all()
+        return Document.objects.all()
 
 
 class FileSerializer(ModelSerializer):
     file = serializers.FileField(write_only=True)
     filename = serializers.SerializerMethodField()
-    size = serializers.IntegerField(read_only=True, source='file.size')
-    owner = ResourceRelatedField(read_only=True)
-
     links = serializers.SerializerMethodField()
+    owner = ResourceRelatedField(read_only=True)
+    size = serializers.IntegerField(read_only=True, source='file.size')
```

Comment:
I think the links method for files should just return a link to the file. ImageSerializers should have overriden links method, that is the same as the method below.

Reply:
Done